modules/chargebee-integration/Makefile

```diff
@@ -22,10 +22,11 @@ ifeq ($(USE_BUILD),true)
 endif
 COMMIT_SHA = $(shell git rev-parse HEAD)
 IMG_SHA_TAG ?= $(VERSION)-$(COMMIT_SHA)
+LDFLAGS = -ldflags="-X main.ver=$(VERSION)"
```

Comment:
It can be configuring only for the build step

Reply:
fixed